app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -116,7 +116,7 @@ private fun WallpaperGroupHeading(
     // and descriptions so they could be localized.
     // In the future, we may want to either use the dynamic wallpaper properties with localized fallbacks
     // or invest in a method of localizing the remote strings themselves.
-    if (collection.name == "classic-firefox") {
+    if (collection.name == "classic-firefox" || collection.name == "default") {
```

Comment:
This bug will actually also be fixed by my changes in https://github.com/mozilla-mobile/fenix/pull/27018. I don't think it would harmful to keep this check for sanity, but it also shouldn't be necessary. I'll leave it up to you